website/source/docs/commands/build.html.md

```diff
@@ -50,6 +50,9 @@ that are created will be outputted at the end of the build.
 -   `-parallel=false` - Disable parallelization of multiple builders (on by
     default).
 
+-   `-timestamp=true` - Enable timestamps for build logs without adding the extra
+    information included if PACKER_LOG is true.
```

Comment:
The phrasing & naming of the argument could be a bit better here; how about something like:
```
-   `-timestamp-ui=true` - Prefix each ui output with an RFC3339 time ( off by default ).
```

Reply:
I'd also put the same content in the command line argument help in `command/build.go`